### ISSUE TITLE
Trying to use tutorial from GE web site and cannot find subdir 

### ISSUE THREAD

**Issue by ederome** [superconductive/ge_tutorials#15]
Hello,
    From the Great Expectations web site here : https://docs.greatexpectations.io/en/stable/guides/tutorials/getting_started/initialize_a_data_context.html
I'm trying to run these steps, but seems that subdir "ge_getting_started_tutorial" does not exists ? 
git clone https://github.com/superconductive/ge_tutorials
cd ge_tutorials/ge_getting_started_tutorial
docker-compose up --detach

**Comment by eugmandel** [comment]
@ederome You are right - thank you for spotting this! Since 0.13 releases contain both the stable (V2) and experimental (V3) versions of the API, we replaced `ge_getting_started_tutorial` directory with two new directories: `getting_started_tutorial_final_stable_api` and `getting_started_tutorial_final_experimental_api`, to give users the choice of runing the Getting Started tutorial in the stable or in the experimental version of the API. We need to update the doc.

**Comment by marclamberti** [comment]
That's great @eugmandel, however, I don't find the docker compose file as given in the doc here https://docs.greatexpectations.io/en/0.13.8/guides/tutorials/getting_started/initialize_a_data_context.html#set-up-your-machine-for-the-tutorial
Any idea?
Thanks
